src/qt/trafficgraphwidget.h

```diff
@@ -37,7 +39,7 @@ public Q_SLOTS:
 
     QTimer *timer;
     float fMax;
-    int nMins;
+    std::chrono::minutes nMins;
```

Comment:
```suggestion
    std::chrono::minutes m_range;
```

Reply:
Addressed in 9bb48bf657c7f2d36309807fbc7b55d0ab581a76